src/switch_jitterbuffer.c

```diff
@@ -332,6 +362,51 @@ static inline void hide_nodes(switch_jb_t *jb)
 	switch_mutex_unlock(jb->list_mutex);
 }
 
+static inline switch_bool_t packet_vad(switch_jb_t *jb, switch_rtp_packet_t *packet, switch_size_t len) {
+	void* payload;
+	uint16_t payload_len = len;
+	uint16_t seq = ntohs(packet->header.seq);
+
+	if (packet->ebody) {
+		payload = packet->ebody;
+	} else {
+		payload = packet->body;
+	}
+	if (payload && payload_len > 0) {
+		return (jb->codec->implementation->vad(payload, payload_len, seq));
+	}
+	return SWITCH_TRUE;
+}
+
+static inline switch_bool_t drop_seq_inactive(switch_jb_t *jb, uint32_t seq)
```

Comment:
Seems like this function is not called from anywhere.

Reply:
indeed, thank you !

I just fixed that mistake and made a few more user tests with netem, to make sure everything is working as expected after that refactoring.

```
sudo tc qdisc add dev wlp0s20f3 root netem delay 175ms 600ms
sudo tc qdisc del dev wlp0s20f3 root netem delay 175ms 600ms
```

```
{
  "size_max_ms": 720,
  "size_est_ms": 20, 
  "acceleration_ms": 1580,
  "expand_ms": 1620,
  "jitter_max_ms": 377,
  "jitter_est_ms": 0,
  "reset": 7,
  "reset_too_big": 0
}
```

I also included the lua script I am using the print/report the jitter buffer stats at the end of a bridged call.